spec/package-spec.coffee

```diff
@@ -205,3 +205,28 @@ describe "Package", ->
 
     it "uses the package name defined in package.json", ->
       expect(metadata.name).toBe 'package-with-a-totally-different-name'
+
+  describe "the initialize() hook", ->
+    it "gets called when the package is activated", ->
+      packagePath = atom.project.getDirectories()[0].resolve('packages/package-with-deserializers')
+      pack = buildPackage(packagePath)
+      spyOn(pack, 'initializeIfNeeded').andCallThrough()
```

Comment:
I thought maybe it would be good to make sure that it wasn't called when loading the main module…but I could remove it or break it out into another test too.